pkg/cluster/internal/providers/docker/provider.go

```diff
@@ -58,6 +58,10 @@ func (p *Provider) Provision(status *cli.Status, cluster string, cfg *config.Clu
 		return err
 	}
 
+	if err := ensureNetwork(fixedNetworkName); err != nil {
+		return errors.Wrap(err, "failed to ensure docker network")
```

Comment:
should we fall back to the default docker network?

Reply:
No?
The default network has no embedded DNS and docker had had user mode networks for a very long time.

Reply:
I'm curious about the DNS behavior with IPv6, seems we are going to have some fun :partying_face:

Reply:
that's true, if embedded DNS misbehaves on IPv6 we may have to use the bridge for IPv6 singlestack :grimacing:

Reply:
I have some thoughts about how we'd make that work, but I hope we don't need to and we can deal with podman + ipv6 singlestack later.